cocnept-designs/basic.css

```diff
@@ -29,3 +53,6 @@ a {
 :disabled {
     opacity: 0.5;
 }
+:focus {
+    outline: none;
```

Comment:
Oof; we could use `:focus-visible` (or a polyfill, or implement our own) for MAXIMUM KEYBOARD ACCESSIBILITY